jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunConfiguration.kt

```diff
@@ -236,6 +236,7 @@ class SamRunSettingsEditor(project: Project) : SettingsEditor<SamRunConfiguratio
         view.regionSelector.selectedRegion = ProjectAccountSettingsManager.getInstance(project).activeRegion
 
         view.credentialSelector.setCredentialsProviders(credentialManager.getCredentialProviders())
+        view.credentialSelector.setSelectedInvalidCredentialsProvider(ProjectAccountSettingsManager.getInstance(project).activeCredentialProvider)
```

Comment:
don't know that this is what we want to call, didn't even know this method existed - but presumably we want to select a 'valid' credential....

Reply:
I'm confused with the method name as well. But this is what we are currently [using for default setting](https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunConfiguration.kt#L259-L267). And this is simply [setting](https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/ui/CredentialProviderSelector.kt#L42-L44) the `selectedItem`.

Reply:
can we rename the method to `setSelectedCredentialsProvider` in that case (had a conversation with @abrooksv )

Reply:
Done.